x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.test.tsx

```diff
@@ -4,124 +4,112 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import '../../../__mocks__/kea.mock';
-import '../../../__mocks__/react_router_history.mock';
+import '../../../__mocks__/shallow_useeffect.mock';
+import { rerender } from '../../../__mocks__';
+import { setMockValues, setMockActions } from '../../../__mocks__/kea.mock';
 
 import React from 'react';
-import { act } from 'react-dom/test-utils';
-import { shallow, ReactWrapper } from 'enzyme';
-
-import { mountAsync, mockHttpValues, setMockValues } from '../../../__mocks__';
+import { shallow, ShallowWrapper } from 'enzyme';
 
 import { LoadingState, EmptyState } from './components';
 import { EnginesTable } from './engines_table';
 
 import { EnginesOverview } from './';
 
 describe('EnginesOverview', () => {
```

Comment:
I left the actual test blocks of this component as-is, I just updated the mocks and assertions to match the new Kea usage.